karma/module.py

```diff
@@ -418,6 +418,19 @@ async def karma_vote(
             + "."
         )
 
+        if sum(votes) < voter_limit:
```

Comment:
Have you tested this? Shouldn't this be `sum(votes.values())`? These kinds of operations usually use `.keys()` implicitly.

Reply:
You are right, it should be votes.valuse() for dictionary. Fixed, squashed, force pushed.